types/rc-calendar/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for rc-calendar 9.6
```

Comment:
I'd like to put this file under the root of the project.

Reply:
@yesmeck Thank you review 👍 

```console
        renamed:    types/rc-calendar/index.d.ts -> index.d.ts
        renamed:    types/rc-calendar/rc-calendar-tests.tsx -> rc-calendar-tests.tsx
        renamed:    types/rc-calendar/tsconfig.json -> tsconfig.json
        renamed:    types/rc-calendar/tslint.json -> tslint.json
```

Do I need to move the file as follows?
If that's okay add the path to package.json `files`

Reply:
Yes, please just put these files under the root of the project.